robot/EDA/tests/browser/settings/eda_settings_course_connections_edit_mode.robot

```diff
@@ -7,22 +7,36 @@ Library         cumulusci.robotframework.PageObjects
 Suite Setup     Open Test Browser
 Suite Teardown  Capture screenshot and delete records and close browser
 
+Test Setup      Run keywords
+...             Go to EDA settings tab          Course Connections      AND
+...             Update enable cc to default
+
 *** Test Cases ***
 Validate Edit Mode For Course Connections, Settings
-    Go to EDA settings tab                  Course Connections
-
-    #Go into Edit Mode
-    Click Button                            Edit
-
-    #Verify that the checkbox can be modified.  
-    Wait for Locator                        eda_settings.cc_checkbox
-    Click on Element                        eda_settings.cc_checkbox
-
-    ${student_select} =  Get Eda Locator    eda_settings.student_select
-    ${faculty_select} =  Get Eda Locator    eda_settings.faculty_select
-
-    Select From List By Label       ${student_select}   Student
-    Select From List By Label       ${faculty_select}   Faculty
-
-    #Save settings
-    Click Button                                Save
+    [Documentation]     This test case validates the settings tab in course connections in edit mode
+    ...                 Also validates the field values of Enable course connections, Default Active
+    ...                 Student Record Type and Default Faculty Record Type are reatined upon saving
+    ...                 which also includes the validation of fields in non edit mode.
+    [tags]                                      unstable        W-041783                W-041784
+    Click edit on EDA settings page
+    Verify enable course connections warning    true
+    Set enable course connections
+    Verify enable course connections warning    false
+    Verify dropdown values                      Default Active Student Record Type
+    ...                                         Default
+    ...                                         Faculty
+    ...                                         Student
+    Verify dropdown values                      Default Faculty Record Type
+    ...                                         Default
+    ...                                         Faculty
+    ...                                         Student
+    Click action button on EDA settings page    Cancel
+    Click edit on EDA settings page
+    Set enable course connections
+    Update dropdown value                       Default Active Student Record Type      Faculty
+    Update dropdown value                       Default Faculty Record Type             Student
```

Comment:
if update_dropdown_value keyword accepts **kwargs you can send 

update dropdown value
...             Default Active Student Record Type=Faculty
...             Default Faculty Record Type=Student

Reply:
@rjanjanam Updated to use kwargs